cyfronet-fid/marketplace
# [#1306] implement xGUS helpdesk requests

# What is in this PR?
This PR contains changes for making technical issues tickets in the xGUS instance directly from mp
(form is accessible from the "Report technical issue" button at right-bottom side of the page).

# How to test it?
1. Fill the form of technical issue report and click send
2. Go to the https://train-ars.ggus.eu/arsys/ and login with the provided credentials
3. From `Quick links` choose `XGUS:HelpDesk`
4. Click on `Search` to see list of tickets
5. Check if ticket is created
